plugins/parcelport/rma_memory_region.hpp

```diff
@@ -6,11 +6,13 @@
 
 #pragma once
 
-#include <hpx/traits/rma_memory_region_traits.hpp>
 #include <plugins/parcelport/parcelport_logging.hpp>
 //
 #include <memory>
 //
+
+#include "rma_memory_region_traits.hpp"
```

Comment:
This could at the very least be included like the other parcelport headers: `#include <plugins/parcelport/libfabric/rm_memory_region_traits.hpp>`. Eventually the parcelports could have a similar structure to the modules with a `src` and `include` directory, but that's outside the scope of this PR.